genesis-builder/src/main.rs

```diff
@@ -55,6 +55,7 @@ static APTOS_PACKAGE_PATHS: Lazy<Vec<PathBuf>> = Lazy::new(|| {
         MOVED_FRAMEWORK_DIR.join("eth-token"),
         MOVED_FRAMEWORK_DIR.join("evm"),
         MOVED_FRAMEWORK_DIR.join("l2"),
+        MOVED_FRAMEWORK_DIR.join("erc20-api"),
```

Comment:
The folder name would be just `erc20`

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/307/commits/82ad18b01634c92e1bde1af47ed72117acb9652b